src/main/java/org/soniakbew/validators/ClientValidator.java

```diff
@@ -0,0 +1,30 @@
+package org.soniakbew.validators;
+import org.soniakbew.daos.ClientDao;
+import org.soniakbew.exceptions.Entity;
+import org.soniakbew.exceptions.InvalidException;
+import org.soniakbew.models.Client;
+import org.soniakbew.models.ClientRequest;
+import java.sql.SQLException;
+import java.util.List;
+
+public class ClientValidator {
+    ClientDao clientDao = new ClientDao();
```

Comment:
can we inject this instead? seems wasteful to create a new instance in validator